<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recycle and Repair Shop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inventoryTab">
       <attribute name="title">
        <string>Inventory</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="inventoryTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addItemButton">
            <property name="text">
             <string>Add Item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editItemButton">
            <property name="text">
             <string>Edit Item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteItemButton">
            <property name="text">
             <string>Delete Item</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="salesTab">
       <attribute name="title">
        <string>Sales</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="barcodeInput">
          <property name="placeholderText">
           <string>Enter barcode...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sellButton">
          <property name="text">
           <string>Sell Item</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="repairTab">
       <attribute name="title">
        <string>Repair Requests</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="repairTableView"/>
        </item>
        <item>
         <widget class="QPushButton" name="newRepairButton">
          <property name="text">
           <string>New Repair Request</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="customersTab">
       <attribute name="title">
        <string>Customers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableView" name="customerTableView"/>
        </item>
        <item>
         <widget class="QPushButton" name="registerCustomerButton">
          <property name="text">
           <string>Register New Customer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>